admin-writer/src/main/resources/raw_data/db/migration/V12__Test_Duration.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE raw_data.test_launches
+ADD COLUMN IF NOT EXISTS duration BIGINT DEFAULT NULL;
```

Comment:
BIGINT is an overkill, integer is sufficient
https://www.postgresql.org/docs/current/datatype-numeric.html

Reply:
I agree that for milliseconds integer will be enough for a test execution time. It's just that the duration of time is usually measured in a long type in Java and a cast to int type will be required